crypto/bio/bss_dgram.c

```diff
@@ -480,7 +480,7 @@ static long dgram_get_mtu_overhead(bio_dgram_data *data)
         {
 #  ifdef IN6_IS_ADDR_V4MAPPED
             struct in6_addr tmp_addr;
```

Comment:
Please can you insert a blank line after this declaration. I know this is how it was before you touched it, but the style checker is complaining.